.woodpecker.yml

```diff
@@ -12,11 +12,18 @@ pipeline:
         #- git fetch --tags
       - git submodule init
       - git submodule update --recursive --remote
+
   check_formatting:
     image: tmknom/prettier
     commands:
       - prettier -c src
 
+  check_typos:
+    image: rust:1.70-slim-buster
+    commands:
+      - cargo install typos-cli@1.14.3
+      - typos src/en
```

Comment:
Would be much faster to download a prebuilt binary.

https://github.com/crate-ci/typos/releases

Reply:
I took your suggestion and am using a prebuilt binary. Let me know further thoughts.

I'd recommend using squash and merge on github but I can manually squash if you don't want to do that.